### ISSUE TITLE
Clicking on expansion arrows does nothing

### ISSUE THREAD

**Issue by ffxsam** [node-inspector/node-inspector#695]
![node_inspector](https://cloud.githubusercontent.com/assets/12532733/8738334/92c4c17e-2be2-11e5-9b0d-789439f2184d.png)

No matter how much I click on these or bash my head against the wall, the arrows do nothing. I'm using the latest version on the latest version of Chrome.

**Comment by ericterpstra** [comment]
Yes, I have the same problem. In Firefox and IE11, I only see the _sources_ and _console_ tabs with  a blank white screen.  Clicking on the arrows results in:

`Uncaught TypeError: window.getComputedStyle(...).getPropertyCSSValue is not a function`

This seems to be a regression of #478 (unless it was never actually fixed in the first place)

**Comment by ericterpstra** [comment]
Actually, never mind. This issue is specific to Meteor, which is bundling an old version of node-inspector. Patching `treeoutline.js` fixes it as ouline in this commit:
https://github.com/node-inspector/node-inspector/commit/92c35ed9fd7e4abdc3a0a8dd818b63d40a9f85e4
